### PR TITLE
Update a README.md

### PR DESCRIPTION
**Pull request by pedrorgleite**
Added a semicolon in the PYTHONPATH so it finds the python modules.

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [ ] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [ ] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

**Review by markomarkusi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -100,8 +100,9 @@ These paths should include wherever the `cmake`, `make`, `arm-none-eabi-gcc`, an
 
 Create a new enviroment variable under your user variables named `PYTHONPATH`. Set the value of the variable to path for the root directory of your fork:
 ```
```
Comment (pasitlaothamatas) [inline README.md:102]: can we leave a note here specifying that the semicolon must be added?

Comment (pasitlaothamatas) [reply README.md:101]: G r a c i a s

**Comment by pasitlaothamatas** [comment]
@gtaharaedmonds anything else you would like to add to the readme after setting up today?

**Review by gtaharaedmonds — APPROVED** [review]
java 11 and python path semicolon were the only fixes I needed, lgtm

File: README.md [hunk 2]
```diff
@@ -44,8 +44,7 @@ sudo apt-get install openocd
   * ARM GNU Embedded Toolchain: https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads (Check `install_gcc_arm_none_eabi.sh` for which version to download)
   * STM32CubeMX: https://www.st.com/en/development-tools/stm32cubemx.html
   * MinGW: https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/mingw-w64-install.exe/download (**select 32-bit verison/i686 architecture**)
-  * Java: https://www.java.com/download/ie_manual.jsp
-
+  * Java: https://www.oracle.com/ca-en/java/technologies/javase/jdk11-archive-downloads.html
```
Comment (pasitlaothamatas) [inline README.md:47]: link here may change. can you add a small note specifying that we get java 11? @pedrorgleite

**Comment by pasitlaothamatas** [comment]
> java 11 and python path semicolon were the only fixes I needed, lgtm

@gtaharaedmonds  anything you want to add here regarding the Clion version to download? i recall you were having some issues with ninja

**Comment by pasitlaothamatas** [comment]
@Eddy13013 anything you would like to add after setting up yesterday? (modifying .env file for windows, adding stuff to syspath, etc.)

**Comment by Eddy13013** [comment]
Yeah, so python paths need to be added to the system Path variable NOT the USER path variable. Add J-Link download to windows install section. Under build, execution, and deployment  --> Cmake --> embedded - change generator from Default/Ninja to Unix Make Files.

**Comment by alxshev** [comment]
@pedrorgleite can you rebase this off the commit and submit the PR again? That should fix the dry run issue and we can merge this in. Smt like:
```
git pull --rebase origin pedrorgleite-readme
git commit
git push
```

**Comment by alxshev** [comment]
TODO: Add Zadig OpenOCD installation description

**Comment by pasitlaothamatas** [comment]
Lets merge this in asap. @pedrorgleite can you talk to people who have set up on windows to see what needs to get added to the readme? Lets try to get this in by saturday

**Comment by timbitss** [comment]
inactive

**Pull request closed without merging**